mypy/main.py

```diff
@@ -513,6 +547,16 @@ def add_invertible_flag(flag: str,
     if options.quick_and_dirty:
         options.incremental = True
 
+    # try setting a valid Python executable based on a specified version
+    if options.python_version:
+        if not options.python_executable:
+            options.python_executable = _python_executable_from_version(options.python_version)
+
+    # Set Python version if given Python executable, but no version
+    if options.python_executable:
+        if not options.python_version:
```

Comment:
May as well combine these with `and` (and above)

Reply:
Nevermind, looks like `python_version` has a default so that can never happen anyway